packages/components/resource-pages/ResourcePopOver.tsx

```diff
@@ -29,50 +45,128 @@ const ResourcePopOver: React.FC<ResourcePopOverProps> = ({
   );
 };
 
+export const BucketClassPopOver: React.FC<BucketClassPopOverProps> = ({
```

Comment:
`BucketClassPopOver` and `DataResourcesPopOver` are exactly same, we can create one component `MCGResourcePopOver` and use that, wdyt ?

Reply:
i think it's better to keep is separate as we have resource URL for redirection which is different for both

Reply:
you can pass it as a prop...
just because 1 line is different we can not have repetition of 20 lines.